### PR TITLE
Support for more shortened urls

### PR DESCRIPTION
**Pull request by n2468txd**
because goo.gl is not enough ...

powered by longurl api v2

### CODE REVIEW

File: lib/spam.ex [hunk 1]
```diff
@@ -34,6 +36,15 @@ defmodule Twitchbot.Spam do
     Agent.update(__MODULE__, fn _set -> Enum.into(bl, HashSet.new) end)
   end
 
+  def update_shortened_urls() do
+    shortened_urls = HTTPoison.get!("http://api.longurl.org/v2/services?format=json", ["User-Agent": "XikBot/2.0"]).body # (Use XikBot v2 useragent as recommended by longurl api)
```
Comment (michaelowens) [inline lib/spam.ex]: Change this to use the username from the config, so if people run it will be their own user agent

**Comment by michaelowens** [comment]
Actually not sure if I'm going to use this. API is giving me timeouts already...

**Comment by n2468txd** [comment]
Hmm, that's unfortunate. We either have to find another API or just use googl.

Just close this PR if it isn't going to work out.

**Comment by n2468txd** [comment]
I've changed the expansion service to use urlex.org, hopefully this is a better API? Alternatively, there should be a way to get the shortened url, and we can use HTTPoison (or similar) to expand and get the referred URL on client-side instead of relying on an API.

Not that this really matters of course, as goo.gl is the main culprit :P

**Comment by n2468txd** [comment]
Actually, while on that thought, found a way to do it locally.

Not sure if that is the best idea as the target may be able to log the IP XikBot is running from?

If it is not the best idea maybe I should revert the commit to the urlex version.

**Comment by michaelowens** [comment]
Definitely not going to resolve them myself due to slow hosts/time outs etc. That would make it timeout too late and be completely useless. I expect google's api to get it from their db, it seemed fast. We'll have to see how spam progresses before making new changes I think.

**Comment by michaelowens** [comment]
Going to close this for now as there is no intention to do this at this point.

**Pull request closed without merging**